packages/shared/src/components/FeedEmptyScreen.tsx

```diff
@@ -35,11 +34,13 @@ function FeedEmptyScreen(): ReactElement {
           onClick={() => {
             const feedName = getFeedName(router.pathname);
 
-            if (feedName === SharedFeedPage.Custom && router.query?.slugOrId) {
-              router.replace(`${webappUrl}feeds/${router.query.slugOrId}/edit`);
-            } else {
-              openModal({ type: LazyModal.FeedFilters });
```

Comment:
This modal and stuff can be removed now as well right?

Reply:
yes, just leaving for a bit until we are done